File: cpp_e9zMZ/cpp/06-thaphinh/84.cpp
#include <iostream>

#include "zvn.h"
using namespace std;
int main() {
    int height = 6;
    string shape = myRepeat(" ", height - 1) + "1\n";
    for (int i = 2; i <= height; i++) {
        string number = to_string(i);
        string space = myRepeat(" ", height - i);
        string spaceCenter = myRepeat(" ", 2 * i - 3);
        shape += space + number + spaceCenter + number + "\n";
    }
    cout << shape << endl;
}

File: python_ToSXs/python/06-thaphinh/84.py
height = 4

shape = " " * (height - 1) + "1" + "\n"
record = ""

for i in range(2, height + 1):
    space = " " * (height - i)
    number = str(i) + " " * (2 * i - 3) + str(i)

    record = space + number + space
    shape += record + "\n"

print(shape)
